src/targets/java/unirest.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var util = require('util');
+
+module.exports = function(options) {
+	var opts = util._extend({
+		indent : '			'
+	}, options);
+
+	var code = [];
+
+	code.push(util.format(
+			'HttpResponse<JsonNode> jsonResponse = Unirest.%s("%s")', this.source.method.toLowerCase(),
```

Comment:
please don't use `Unirest.METHOD`, it needs to support ANY method name, not just the Unirest predefined methods